jinja2/runtime.py

```diff
@@ -313,12 +313,7 @@ def __repr__(self):
         )
 
 
-# register the context as mapping if possible
-try:
-    from collections import Mapping
-    Mapping.register(Context)
-except ImportError:
-    pass
```

Comment:
I'm not sure why this `ImportError` handling was here, seeing that Python 2.6 always has `collections.Mapping` available. It was added a long time ago (>10 years) in 9bb7e4779182490abc6e1784b0ee63d22b91b11e, so I suppose that was needed for an older Python version but isn't anymore.